__tests__/readers.test.js

```diff
@@ -0,0 +1,187 @@
+// In Sequelize, all of our models will be exported from src/models/index.js
```

Comment:
All the comments from this file should be removed as none of them are adding anything to the value or understanding of the code, especially the console logs